app/build.gradle

```diff
@@ -102,12 +102,10 @@ android {
 dependencies {
 	implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlinVersion"
 
-	implementation 'com.google.android.material:material:1.3.0'
-	implementation 'androidx.appcompat:appcompat:1.3.0'
+	implementation 'com.google.android.material:material:1.7.0'
+	implementation 'androidx.appcompat:appcompat:1.5.1'
 	implementation 'androidx.activity:activity:1.2.3'
-	implementation('androidx.fragment:fragment:1.1.0!!') {
-		because 'issues with MapBox SDK onDestroy after upgrading to 1.2.0-alpha02 or newer versions'
-	}
```

Comment:
I think the ANR probably comes from this update. When I reverted back to `fragment:1.1.0`, the freeze is gone.

Reply:
If we keep the `com.google.material` library at version 1.5.0 (which is around 1.5 years old currently), we don't need to update the `androidx.fragment` dependency. I guess that's just the best way for now, even the latest version of `androidx.fragment` caused the same issues.

Reply:
Cool, thanks for looking into this.